pkg/pillar/cmd/volumemgr/handlevolumeref.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package volumemgr
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	log "github.com/sirupsen/logrus"
+)
+
+func handleVolumeRefModify(ctxArg interface{}, key string,
+	configArg interface{}) {
+
+	log.Infof("handleVolumeRefModify(%s)", key)
+	config := configArg.(types.VolumeRefConfig)
+	ctx := ctxArg.(*volumemgrContext)
+	vs := lookupVolumeStatus(ctx, config.VolumeKey())
+	if vs != nil {
+		vs.RefCount++
```

Comment:
If handleVolumeRefModify is called due to some random change (as opposed to a create), why would you add refcount? How about adding a separate handleVolumeRefCreate function and have it do the refcount work?

Reply:
If vs is nil I think you should still create a VolumeRefStatus with State = INIT and config.RefCount so that zedmanager can see there is some forward motion. Otherwise you have to create it when the VolumeStatus is created which would be odd.